### ISSUE TITLE
Migrate custom reporter and benchmark for ginkgo v2 in GPATs

### ISSUE THREAD

**Issue by kieron-dev** [cloudfoundry/garden-performance-acceptance-tests#47]
We saw the following deprecation notice after recently upgrading ginkgo to 1.16.4:

```
You're using deprecated Ginkgo functionality:
=============================================
Ginkgo 2.0 is under active development and will introduce (a small number of) breaking changes.
To learn more, view the migration guide at https://github.com/onsi/ginkgo/blob/v2/docs/MIGRATING_TO_V2.md
To comment, chime in at https://github.com/onsi/ginkgo/issues/711

  You are using a custom reporter.  Support for custom reporters will likely be removed in V2.  Most users were using them to generate junit or teamcity reports and this functionality will be merged into the core reporter.  In addition, Ginkgo 2.0 will support emitting a JSON-formatted report that users can then manipulate to generate custom reports.

  If this change will be impactful to you please leave a comment on https://github.com/onsi/ginkgo/issues/711
  Learn more at: https://github.com/onsi/ginkgo/blob/v2/docs/MIGRATING_TO_V2.md#removed-custom-reporters
  Measure is deprecated and will be removed in Ginkgo V2.  Please migrate to gomega/gmeasure.
  Learn more at: https://github.com/onsi/ginkgo/blob/v2/docs/MIGRATING_TO_V2.md#removed-measure
    /tmp/build/b0514c9e/src/code.cloudfoundry.org/garden-performance-acceptance-tests/create_test.go:23

To silence deprecations that can be silenced set the following environment variable:
  ACK_GINKGO_DEPRECATIONS=1.16.4
```

We added the ACK_GINKGO_DEPRECATIONS to the calling script, but this should be fixed properly once ginkgo v2 is available.

**Comment by gcapizzi** [comment]
Blocked on the release of Ginkgo v2.

**Comment by MarcPaquette** [comment]
Ginkgo v2 is now unblocked and this work should begin.

**Comment by MarcPaquette** [comment]
Closing out this issue as this work is now complete
